metagraph/integration_tests/test_align.py

```diff
@@ -366,7 +363,7 @@ def test_simple_align_edit_distance_all_graphs(self, representation):
 
         params = self._get_stats(self.tempdir.name + '/genome.MT' + graph_file_extension[representation])
         self.assertEqual('11', params['k'])
-        self.assertEqual('16461', params['nodes (k)'])
+        self.assertEqual('16438', params['nodes (k)'])
```

Comment:
This number decreases if `valid_edges` is generated and serialized for DBGSuccinct. Probably because dummy nodes are excluded this way. Same with the change above.